src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -125,7 +125,7 @@ public function handle(AddProductToOrderCommand $command)
             $cart,
             $order->getCurrentOrderState(),
             $cart->getProducts(),
-            $command->getOrderInvoiceId() ?: 0
+            $invoice->id ?: 0
```

Comment:
I tried to add a new product with new invoice and I get an exception from the ajax call `Notice: Trying to get property 'id' of non-object`
I think it's because I tried it on an order that already has an invoice, then `createNewOrEditExistingInvoice` returns null and no invoice is created.. Not sure it's the expected behaviour But if it is then you should check use `$invoice` is null

Reply:
```suggestion
            !empty($invoice->id) ? $invoice->id : 0
```
After more tests it seems to happen only on orders that have no invoice (in which case the behaviour in `createNewOrEditExistingInvoice` makes total sense)